### PR TITLE
[IMP]10729 add module hide_purchase_price_version_in_sales.

### PR DESCRIPTION
**Pull request by Rona111**

### CODE REVIEW

**Comment by elicoidal** [comment]
Headers/README

**Comment by Rona111** [comment]
cc @elicoidal @duanyp1991 @siyuancn @seb-elico 
I finished the test of migration,and fix the  RADME/openerp.py/PEP8.

File: hide_purchase_price_version_in_sales/price_list.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# -*- coding: utf-8 -*-
+# © 2016 Elico Corp (www.elico-corp.com).
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp.osv import fields, osv
+from openerp.tools.translate import _
+
+
+class product_pricelist_version(osv.osv):
+    _inherit = "product.pricelist.version"
```
Comment (siyuancn) [inline hide_purchase_price_version_in_sales/price_list.py:10]: what's the usage of the inherit?

**Comment by elicoidal** [comment]
LGTM

**Pull request merged**